swagger.yaml

```diff
@@ -4420,7 +8696,7 @@ definitions:
   connectivity-report:
     type: object
     properties:
-      remote_hosts:
+      remote_agents:
```

Comment:
we should have 2 definitions, with 'remote_hosts' for v1 and with 'remote_agents' for v2

Reply:
fixed